doc/Language/faq.pod6

```diff
@@ -362,49 +362,15 @@ multi postcircumfix:<[- ]> (Str:D $str is copy, +@indices) {
 say '0123456789'[- 1..3, 8 ];  # OUTPUT: «045679␤»
 =end code
 
-=head2 String: How can I split a string in equal parts
+=head2 String: How can I split a string in equal parts?
 
 L<.comb|/routine/comb> is accepting an optional L<Int|/type/Int>:
 
 =begin code
 .say for 'abcdefghijklmnopqrstuvwxyz'.comb: 8;  # OUTPUT: «abcdefgh␤ijklmnop␤qrstuvwx␤yz»
 =end code
 
-=head2 List: How can I remove duplicates items in a list
-
-Use L<.unique|/routine/unique>:
-
-=begin code
-(1, 2, 3, 2, 3, 1, 1, 0).unique;  # OUTPUT: «(1 2 3 0)␤»
-=end code
-
-=head2 List: How can I delete an element from a list
-
-To delete by index, use L<.splice|/routine/splice>:
-
-=begin code
-my @a = <3 18 4 8 92 14 30>;
-say @a.splice(3, 1);  # OUTPUT: «[8]␤»
-say @a;               # OUTPUT: «[3 18 4 92 14 30]␤»
-=end code
-
-Note that the L<delete subscript|/language/subscripts#index-entry-:delete_(subscript_adverb)> replaces the element with its natural (i.e. default) state. Here (Any):
-
-=begin code
-my @a = <3 18 4 8 92 14 30>;
-@a[3]:delete;
-say @a;  # OUTPUT: «[3 18 4 (Any) 92 14 30]␤»
-=end code
-
-To delete by value, use L<.grep|/routine/grep>:
-
-=begin code
-my @letters = <a b c>;
-@letters .= grep: * ne "b";
-say @letters;  # OUTPUT: «[a c]␤»
-=end code
-
-=head2 List: How can I show index while looping through a list
+=head2 List: How can I show index while looping through a list?
 
 Use L<.kv|/routine/kv> or L<.pairs|/routine/pairs>:
```

Comment:
Again, this is a direct use of the .kv method. Increase visibility by adding another example, but it's not a FAQ.